src/ui-test/tests/debugger.test.ts

```diff
@@ -54,15 +63,20 @@ describe('Camel Debugger tests', function () {
         await new EditorView().closeAllEditors();
     });
 
-    it('Toogle breakpoint on log line (14)', async function () {
-        await executeCommand(CAMEL_RUN_DEBUG_ACTION_LABEL);
-        await waitUntilTerminalHasText(driver, TEST_ARRAY_RUN_DEBUG);
+    it('Toggle breakpoint on log line (17)', async function () {
         await driver.wait(async function () {
-            return await new TextEditor().toggleBreakpoint(14);
+            return await new TextEditor().toggleBreakpoint(17);
         }, 5000);
+        breakpointToggled = true;
+        skip = false;
     });
 
     it('Update Body value with Camel debugger', async function () {
+        if (skip){ 
```

Comment:
I don't like it much, but I see the point that other tests are connected and all will fail anyway in case previous ones not pass..

maybe at least visually could also ternary operator looks a bit more elegant WDYT?

`skip ? this.test?skip() : skip = true;`

Reply:
It was recognized by sonar as "CODE_SMELL" :smile: previously

Reply:
ok, sonarcloud is the most smart guy in the room then

Reply:
![Screenshot from 2023-08-01 16-26-35](https://github.com/camel-tooling/camel-dap-client-vscode/assets/46084942/e3f53158-7624-42e1-80e1-81c810300b3b)

Reply:
compromise between the three of us =)

```
if (skip){ this.test?.skip(); }
skip = true;
```

Reply:
I don't have strong opinion here.. choose more readable and clean notation

Reply:
I will use the same as for breakpoint :open_mouth: